packages/drivers/odsp-driver/src/odspDocumentServiceFactory.ts

```diff
@@ -35,3 +40,70 @@ export class OdspDocumentServiceFactory
         );
     }
 }
+
+export class LocalOdspDocumentServiceFactory extends OdspDocumentServiceFactoryCore {
+    constructor(
+        private readonly fluidFile: Uint8Array | string,
+    ) {
+        super(
+            async (_options) => { return null; }, // TODO
+            undefined,
+            async () => getSocketIo(), // TODO
```

Comment:
getSocketIOClient & getWebsocketToken should be easy - it should just throw some exception (maybe also log to telemetry that this call happened) - if everything else is done right, these calls should never happen.

first callback (getStorageToken) is a bit harder - ideally, we also structure code in a way where these calls never made, but I'm not sure if it's possible.